src/main/java/com/codegym/shopyy/configuration/AppConfiguration.java

```diff
@@ -0,0 +1,34 @@
+package com.codegym.shopyy.configuration;
+
+
+import org.modelmapper.ModelMapper;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.autoconfigure.EnableAutoConfiguration;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.web.servlet.config.annotation.CorsRegistry;
+import org.springframework.web.servlet.config.annotation.WebMvcConfigurer;
+
+@Configuration
+@EnableAutoConfiguration
+public class AppConfiguration implements WebMvcConfigurer {
+
+    private final long MAX_AGE_SECS = 3600;
+
+    @Value("http://localhost:5500")
+    private String[] allowedOrigins;
+
+    @Override
```

Comment:
remove it because this code using for Security

Reply:
complete